api/catalog/api/views/image_views.py

```diff
@@ -131,7 +132,13 @@ def watermark(self, request, *_, **__):
         watermarked, exif = watermark(image_url, image_info, params.data["watermark"])
         # Re-insert EXIF metadata.
         if exif:
-            exif_bytes = piexif.dump(exif)
+            # piexif dump raises a struct error when the value is not
+            # between -2147483648 and 2147483647
+            # https://github.com/WordPress/openverse-api/issues/849
+            try:
+                exif_bytes = piexif.dump(exif)
+            except struct.error:
```

Comment:
It looks like the Sentry alert linked in #849 is raising a `InvalidImageDataError` exception, should we be catching that instead?

Reply:
I got really confused with Sentry. It seems like I'm opening the same error (based on its identifier that starts with a80...) using different links, and I'm getting different errors and stack traces. 

So, I added both the struct error and a ValueError (which is the parent of `InvalidImageDataError`): 
https://github.com/hMatoba/Piexif/blob/3422fbe7a12c3ebcc90532d8e1f4e3be32ece80c/piexif/_exceptions.py#L1

Reply:
Ah, that makes sense - perfect!